src/xcode/ENA/ENA/Source/Scenes/RiskLegend/RiskLegendViewController.swift

```diff
@@ -79,7 +79,8 @@ class RiskLegendViewController: DynamicTableViewController {
 			.navigationSubtitle(
 				text: AppStrings.RiskLegend.subtitle,
 				insets: insets,
-				accessibilityIdentifier: AccessibilityIdentifiers.RiskLegend.subtitle),
+				accessibilityIdentifier: AccessibilityIdentifiers.RiskLegend.subtitle,
+				accessibilityTraits: .none),
```

Comment:
Can you make the same change in the StatisticsInfoViewController? I know it's not mentioned in the ticket, because its new, but it's basically a copy of the RiskLegendViewController, so it probably has the same issue. Or maybe we could just change the default value to .none, because it seems that these are the only two places that use `.navigationSubtitle`.

Reply:
I'll check and change it - like the idea.

Reply:
And it also works for the UIImage below, which is another bug ticket.  👍